src/BackupManager/Factory/DatabaseFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace DigipolisGent\Robo\Task\Deploy\BackupManager\Factory;
+
+use BackupManager\Config\Config;
+use BackupManager\Databases\DatabaseProvider;
+use BackupManager\Databases\PostgresqlDatabase;
+use DigipolisGent\Robo\Task\Deploy\BackupManager\Databases\MysqlDatabase;
+
+class DatabaseFactory implements DatabaseFactoryInterface
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function create($dbConfig)
```

Comment:
So we can use this from Drupal7 by parsing the settings file and get the db config from there for initialization of this?